port/unix/omrsysinfo.c

```diff
@@ -4215,6 +4218,51 @@ omrsysinfo_cgroup_is_memlimit_set(struct OMRPortLibrary *portLibrary)
 #endif /* defined(LINUX) && !defined(OMRZTPF) */
 }
 
+intptr_t
+omrsysinfo_cgroup_get_handle_subsystem_file(struct OMRPortLibrary *portLibrary,  uint64_t subsystemFlag, const char *fileName)
+{
+#if defined(LINUX) && !defined(OMRZTPF)
+	int32_t rc = OMRPORT_ERROR_SYSINFO_CGROUP_UNSUPPORTED_PLATFORM;
+	FILE *file = NULL;
+	rc = getHandleOfCgroupSubsystemFile(portLibrary, subsystemFlag, fileName, &file);
+	if ((rc == 0) && (file != NULL)) {
+		return fileno(file);
```

Comment:
This will leak `FILE` handles. It should be using `omrfile_open` and the documentation should discuss the need to call `omrfile_close`.

Reply:
Removed the `FILE` handles leak. Now the `omrsysinfo_cgroup_get_handle_subsystem_file` api returns the file descriptor which is given by the `omrfile_open`. Added the documentation stating that `omrfile_close` should be called after calling this api.